src/enums.h

```diff
@@ -290,7 +290,27 @@ struct tripoint : public JsonSerializer, public JsonDeserializer {
     {
         return tripoint( -x, -y, -z );
     }
-
+    /*** some point operators and functions ***/
+    tripoint operator+(const point &rhs) const
+    {
```

Comment:
This operator is already implementation (see below), but it is not const. Please remove this implementation and add the const to the other one.